### ISSUE TITLE
Loss function for fine-tuning

### ISSUE THREAD

**Issue by austinmw** [FlagOpen/FlagEmbedding#423]
Hi, what is the name of the loss function used for fine-tuning? And could you please point me to where it is defined?

**Comment by staoxiao** [comment]
Hi, the loss function is at https://github.com/FlagOpen/FlagEmbedding/blob/master/FlagEmbedding/baai_general_embedding/finetune/modeling.py#L98

Yes, we use CrossEntropyLoss with In-Batch Negative Sampling. We will use the provided negatives and if set `use_inbatch_neg` to True (default value is True), we also use the passages from other queries as negatives.

**Comment by qcnace** [comment]
how to change the loss function ? can u show me
